protocols/gossipsub/src/peer_score.rs

```diff
@@ -428,6 +437,13 @@ impl PeerScore {
             if peer_stats.behaviour_penalty < params_ref.decay_to_zero {
                 peer_stats.behaviour_penalty = 0.0;
             }
+
+            // decay slow peer score
+            peer_stats.slow_peer_penalty *= params_ref.slow_peer_decay;
```

Comment:
the default value of `slow_peer_decay` is set to `0.2`, which means that every heartbeat the `slow_peer_penalty` is divided by `5`. this seems like a lot, is the logic documented somewhere?

Reply:
cc @AgeManning